lib/elixir/lib/option_parser.ex

```diff
@@ -543,7 +542,12 @@ defmodule OptionParser do
         {[], false}
     end
 
-    {aliases, switches, strict?, allow_nonexistent_atoms?}
+    %{
+      aliases: opts[:aliases] || [],
+      allow_nonexistent_atoms?: opts[:allow_nonexistent_atoms] || false,
```

Comment:
We just rarely use it in key names, I believe, never in public APIs. :man_shrugging: